<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopPet</class>
 <widget class="QMainWindow" name="DesktopPet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DesktopPet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#DesktopPet{border-image: url(:/angry/img/Animations/angry/angry_00.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_head">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cymbal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/cymbal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/eat.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stomach">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>130</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>160</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_drink">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/drink.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scratch">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>700</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/scratch.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rightfoot">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>900</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fart">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/fart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pie">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>75</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Buttons/img/Buttons/pie.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_leftfoot">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>910</y>
      <width>65</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
